vizro-core/docs/pages/user-guides/run.md

```diff
@@ -2,11 +2,22 @@
 
 This guide shows you how to launch your dashboard in different ways. By default, your dashboard apps run on localhost port 8050 so is accessible at [http://127.0.0.1:8050/](http://127.0.0.1:8050/).
 
+## Py.Cafe
+
+You can launch your dashboard and edit the code live on [Py.Cafe](https://py.cafe/).
+
+For this basic example, follow the [Edit live on Py.Cafe](https://py.cafe/maxi.schulz/vizro-tutorial-first-dashboard) link below the dashboard to experiment with the code.
+
+<iframe src="https://py.cafe/stichbury/vizro-iris-viz" width="100%" height="600px"></iframe>
```

Comment:
Should this not then be displayed in the iframe? https://py.cafe/maxi.schulz/vizro-tutorial-first-dashboard